src/Cogworks.CogFlare.Core/Composers/ExternalCachePurgeComposer.cs

```diff
@@ -16,6 +16,7 @@ public void Compose(IUmbracoBuilder builder)
         builder.Services
             .AddTransient(_ => cogFlareSettings)
```

Comment:
Lets use the Options pattern: https://learn.microsoft.com/en-us/dotnet/core/extensions/options

Like:

```csharp
_ = services.Configure<CogFlareSettings>(config.GetSection(nameof(CogFlareSettings)));
```

this will allow to inject `IOptionsMonitor<CogFlareSettings>` and take in real time the latest values.